documentation/modules/managing/con-service-discovery.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// managing/assembly-management-tasks.adoc
+
+[id='proc-add-service-discovery-{context}']
+= Discovering services using labels and annotations
+
+Service discovery makes it easier for client applications running in the same cluster as {ProductName} to interact with a Kafka cluster.
```

Comment:
"the same cluster" - which type of cluster?

Reply:
_Kubernetes cluster_